### PR TITLE
Fix stuff after new update <3

### PR DESCRIPTION
**Pull request by 1337-69420**

### CODE REVIEW

**Comment by ihavezerohealth** [comment]
epic

**Comment by V4NSH4J** [comment]
+1

File: instance/autobuy.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ func (in *Instance) abLaptop(_ discord.Message) {
 	if trigger == nil || trigger.Value != postmemeCmdValue {
 		return
 	}
-	in.sdlr.ResumeWithCommand(&scheduler.Command{
+	in.sdlr.PrioritySchedule(&scheduler.Command{
```
Comment (dankgrinder) [inline instance/autobuy.go:19]: Why did you change this to `PrioritySchedule` instead of `ResumeWithCommand`?

File: instance/autobuy.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ func (in *Instance) abHuntingRifle(_ discord.Message) {
 	if trigger == nil || trigger.Value != huntCmdValue {
 		return
 	}
-	in.sdlr.ResumeWithCommand(&scheduler.Command{
+	in.sdlr.PrioritySchedule(&scheduler.Command{
```
Comment (dankgrinder) [inline instance/autobuy.go:30]: Same here

File: instance/autobuy.go [hunk 3]
```diff
@@ -38,7 +38,7 @@ func (in *Instance) abFishingPole(_ discord.Message) {
 	if trigger == nil || trigger.Value != fishCmdValue {
 		return
 	}
-	in.sdlr.ResumeWithCommand(&scheduler.Command{
+	in.sdlr.PrioritySchedule(&scheduler.Command{
```
Comment (dankgrinder) [inline instance/autobuy.go:41]: And here..

File: instance/router.go [hunk 4]
```diff
@@ -128,7 +128,7 @@ func (in *Instance) router() *discord.MessageRouter {
 		Channel(in.ChannelID).
 		Author(DMID).
 		ContentMatchesExp(exp.search).
-		Mentions(in.Client.User.ID).
+		//Mentions(in.Client.User.ID).
```
Comment (dankgrinder) [inline instance/router.go:131]: Looks good, if you could remove the comment as well to not clutter the code too much

File: instance/router.go [hunk 5]
```diff
@@ -164,7 +164,7 @@ func (in *Instance) router() *discord.MessageRouter {
 			Channel(in.ChannelID).
 			Author(DMID).
 			ContentContains("You don't have a fishing pole").
-			Mentions(in.Client.User.ID).
+			//Mentions(in.Client.User.ID).
```
Comment (dankgrinder) [inline instance/router.go:167]: Remove comment here as well

File: instance/router.go [hunk 6]
```diff
@@ -174,7 +174,7 @@ func (in *Instance) router() *discord.MessageRouter {
 			Channel(in.ChannelID).
 			Author(DMID).
 			ContentContains("You don't have a hunting rifle").
-			Mentions(in.Client.User.ID).
+			//Mentions(in.Client.User.ID).
```
Comment (dankgrinder) [inline instance/router.go:177]: And here

**Review by dankgrinder — CHANGES_REQUESTED** [review]
Also, just like the other PR, would appreciate if you could target the develop branch :)

File: instance/autobuy.go (hunk 3 above)
Comment (1337-69420) [reply instance/autobuy.go:41]: I'll explain, when I was testing stuff, resumewithcommand glitched. Basically it would execute pls search then it would execute pls buy rifle 1 which wouldnt be executed as the bot will say "not a valid search option" thus this change was required

**Comment by 1337-69420** [comment]
I don't know how to edit this particular pull request so I will open up a new one
